src/Analysis/Engine/Impl/DependencyResolution/PathResolverSnapshot.cs

```diff
@@ -14,31 +14,59 @@
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
 
+using Microsoft.PythonTools.Analysis.Infrastructure;
+using Microsoft.PythonTools.Parsing;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Runtime.InteropServices;
-using Microsoft.PythonTools.Analysis.Infrastructure;
-using Microsoft.PythonTools.Parsing;
+using System.Text;
 
 namespace Microsoft.PythonTools.Analysis.DependencyResolution {
     internal partial struct PathResolverSnapshot {
-        private static readonly Node NullRoot = Node.CreateRoot("$");
+        private static readonly Node NullRoot = Node.CreateNullRoot();
         private static readonly bool IgnoreCaseInPaths = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
```

Comment:
`IgnoreCaseInPaths ` - might be worth making something common in `Infrastructure`, prob will be useful in other places.